src/metabase/pulse.clj

```diff
@@ -0,0 +1,34 @@
+(ns metabase.pulse
+  (:require [hiccup.core :refer [html]]))
+
+(def ^:private td-style " font-size: 14pt; text-align: left; padding-right: 1em; padding-top: 8px; ")
+(def ^:private th-style " font-size: 12pt; text-transform: uppercase; border-bottom: 4px solid rgb(248, 248, 248); padding-bottom: 10px; ")
+
+(defn bar-chart-row
+  [index row max-value]
+  [:tr {:style (if (odd? index) "color: rgb(189,193,191);" "color: rgb(124,131,129);")}
+    [:td {:style td-style} (first row)]
+    [:td {:style (str td-style " font-weight: bolder;")} (second row)]
```

Comment:
if you're just using the first two elements of row it arguably makes sense just to destructure it in the args vector itself so you can give them meaningful names instead of repeated calls to `(first row)` and `(second row)`.